src/main/java/reactor/netty/http/client/HttpClientState.java

```diff
@@ -42,6 +42,15 @@ public String toString() {
 			return "[request_sent]";
 		}
 	},
+	/**
+	 * The request is about to be retried
+	 */
+	RETRY_ENABLED() {
+		@Override
+		public String toString() {
+			return "[retry_enabled]";
+		}
+	},
```

Comment:
Could this be called `RETRYING` perhaps? Again `RETRY_ENABLED` sounds like a config time callback.

For source code ordering, it's a bit strange to see this state in the middle of the request related ones. Perhaps it can be ordered at the top of this class since it belongs more with the base class (connection related).

Reply:
done, changed to `RETRYING`